File: jsprobelms/day9/missingprime.js
function primemissing() {
  // find the missing prime in an array
  let arr = [2,7];
  min = arr[0];
  max = arr[0];
  for (i of arr) {
    if (i < min) {
      min = i;
    }
    if (i > max) {
      max = i;
    }
  }
  // console.log(min,max);
  for (i = 1; i < max; i++) {
    fact = 0;
    for (j = 2; j < i; j++) {
      if (i % j == 0) {
        fact++;
        break;
      }
    }
    if (fact == 0 && !arr.includes(i)) {
      console.log(i, "is the missing prime number");
    }
  }
}
primemissing();

File: pythonproblems/day9/missing-prime.py
# find the missing prime in an array
arr=[5,4,8,9,7,10,15]
max=arr[0]
for i in arr:
    if i>max:
        max=i

for j in range(1,max+1):
    fact=0
    for k in range(2,j):
        if j%k==0:
            fact+=1
            break
    if fact==0 and j not in arr:
        print(j,"is the missing prime")

